<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01 Transitional//EN' 'http://www.w3.org/TR/html4/loose.dtd'>
<html>
<head>
  <!-- SchemaSpy rev 590 -->
  <title>SchemaSpy - david_chado.public</title>
  <link rel=stylesheet href='schemaSpy.css' type='text/css'>
  <meta HTTP-EQUIV='Content-Type' CONTENT='text/html; charset=ISO-8859-1'>
  <SCRIPT LANGUAGE='JavaScript' TYPE='text/javascript' SRC='jquery.js'></SCRIPT>
  <SCRIPT LANGUAGE='JavaScript' TYPE='text/javascript' SRC='schemaSpy.js'></SCRIPT>
  <SCRIPT LANGUAGE='JavaScript' TYPE='text/javascript'>
    $(function(){
      associate($('#showTables'), $('.tbl'));
      associate($('#showViews'),  $('.view'));
      jQuery.fn.alternateRowColors = function() {
        $('tbody tr:visible').each(function(i) {
          if (i % 2 == 0) {
            $(this).removeClass('even').addClass('odd');
          } else {
            $(this).removeClass('odd').addClass('even');
          }
        });
        return this;
      };
      $('#showTables, #showViews').click(function() {
        $('table.dataTable').alternateRowColors();
      });
      $('table.dataTable').alternateRowColors();
    })
  </SCRIPT>
</head>
<body>
<table id='headerHolder' cellspacing='0' cellpadding='0'><tr><td>
<div id='header'>
 <ul>
  <li id='current'><a href='index.html' title='All tables and views in the schema'>Tables</a></li>
  <li><a href='relationships.html' title='Diagram of table relationships'>Relationships</a></li>
  <li><a href='utilities.html' title='View of tables with neither parents nor children'>Utility&nbsp;Tables</a></li>
  <li><a href='constraints.html' title='Useful for diagnosing error messages that just give constraint name or number'>Constraints</a></li>
  <li><a href='anomalies.html' title="Things that might not be quite right">Anomalies</a></li>
  <li><a href='columns.byTable.html' title="All of the columns in the schema">Columns</a></li>
  <li><a href='http://sourceforge.net/donate/index.php?group_id=137197' title='Please help keep SchemaSpy alive' target='_blank'>Donate</a></li>
 </ul>
</div>
</td></tr></table>
<div class='content' style='clear:both;'>
<table width='100%' border='0' cellpadding='0'>
 <tr>
  <td class='heading' valign='middle'><span class='header'>SchemaSpy Analysis of <span title='Database'>david_chado</span>.<span title='Schema'>public</span></span></td>
  <td class='heading' align='right' valign='top' title='John Currier - Creator of Cool Tools'><span class='indent'>Generated by</span><br><span class='indent'><span class='signature'><a href='http://schemaspy.sourceforge.net' target='_blank'>SchemaSpy</a></span></span></td>
 </tr>
</table>
<table width='100%'>
 <tr><td class='container'>
<span class='container'>Generated by <span class='signature'><a href='http://schemaspy.sourceforge.net' target='_blank'>SchemaSpy</a></span> on Fri Jun 06 11:12 CDT 2014</span>
 </td></tr>
 <tr>
  <td class='container'>Database Type: PostgreSQL - 9.3.1  </td>
  <td class='container' align='right' valign='top' rowspan='3'>
    <a href='http://sourceforge.net' target='_blank'><img src='http://sourceforge.net/sflogo.php?group_id=137197&amp;type=1' alt='SourceForge.net' border='0' height='31' width='88'></a><br>
    <br><div style="margin-right: 2pt;">
<script type="text/javascript"><!--
google_ad_client = "pub-9598353634003340";
google_ad_channel ="SchemaSpy-generated";
google_ad_width = 234;
google_ad_height = 60;
google_ad_format = "234x60_as";
google_ad_type = "text";
google_color_border = "9bab96";
google_color_link = "489148";
google_color_text = "000000";
//-->
</script>
<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</div>
  </td>
 </tr>
 <tr>
  <td class='container'><br><a href='david_chado.public.xml' title='XML Representation'>XML Representation</a><br><a href='insertionOrder.txt' title='Useful for loading data into a database'>Insertion Order</a>&nbsp;<a href='deletionOrder.txt' title='Useful for purging data from a database'>Deletion Order</a>&nbsp;(for database loading/purging scripts)</td>
 </tr>
</table>
<div class='indent'>
<p><b><label for='showTables'><input type='checkbox' id='showTables' checked>Tables</label> <label for='showViews'><input type='checkbox' id='showViews' checked>Views</label> <label for='showComments'><input type=checkbox checked id='showComments'>Comments</label>
</b>
<table class='dataTable' border='1' rules='groups'>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup class='comment'>
<thead align='left'>
<tr>
  <th valign='bottom'>Table / View</th>
  <th align='right' valign='bottom'>Children</th>
  <th align='right' valign='bottom'>Parents</th>
  <th align='right' valign='bottom'>Columns</th>
  <th align='right' valign='bottom'>Rows</th>
  <th class='comment' align='left' valign='bottom'>Comments</th>
</tr>
</thead>
<tbody>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/acquisition.html'>acquisition</a></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>This represents the scanning of hybridized material. The output of this process is typically a digital image of an array.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/acquisition_relationship.html'>acquisition_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Multiple monochrome images may be merged to form a multi-color image. Red-green images of 2-channel hybridizations are an example of this.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/acquisitionprop.html'>acquisitionprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Parameters associated with image acquisition.</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/all_feature_names.html'>all_feature_names</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/analysis.html'>analysis</a></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>10</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>An analysis is a particular type of a<br>
    computational analysis; it may be a blast of one sequence against<br>
    another, or an all by all blast, or a different kind of analysis<br>
    altogether. It is a single unit of computation.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/analysisfeature.html'>analysisfeature</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Computational analyses generate features (e.g. Genscan generates transcripts and exons; sim4 alignments generate similarity/match features). analysisfeatures are stored using the feature table from the sequence module. The analysisfeature table is used to decorate these features, with analysis specific attributes. A feature is an analysisfeature if and only if there is a corresponding entry in the analysisfeature table. analysisfeatures will have two or more featureloc entries,<br>
 with rank indicating query/subject</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/analysisfeatureprop.html'>analysisfeatureprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/analysisprop.html'>analysisprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/arraydesign.html'>arraydesign</a></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>18</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>General properties about an array.<br>
An array is a template used to generate physical slides, etc.  It<br>
contains layout information, as well as global array properties, such<br>
as material (glass, nylon) and spot dimensions (in rows/columns).</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/arraydesignprop.html'>arraydesignprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Extra array design properties that are not accounted for in arraydesign.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/assay.html'>assay</a></td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>10</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>An assay consists of a physical instance of<br>
an array, combined with the conditions used to create the array<br>
(protocols, technician information). The assay can be thought of as a hybridization.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/assay_biomaterial.html'>assay_biomaterial</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A biomaterial can be hybridized many times (technical replicates), or combined with other biomaterials in a single hybridization (for two-channel arrays).</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/assay_project.html'>assay_project</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Link assays to projects.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/assayprop.html'>assayprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Extra assay properties that are not accounted for in assay.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/biomaterial.html'>biomaterial</a></td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A biomaterial represents the MAGE concept of BioSource, BioSample, and LabeledExtract. It is essentially some biological material (tissue, cells, serum) that may have been processed. Processed biomaterials should be traceable back to raw biomaterials via the biomaterialrelationship table.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/biomaterial_dbxref.html'>biomaterial_dbxref</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/biomaterial_relationship.html'>biomaterial_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Relate biomaterials to one another. This is a way to track a series of treatments or material splits/merges, for instance.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/biomaterial_treatment.html'>biomaterial_treatment</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Link biomaterials to treatments. Treatments have an order of operations (rank), and associated measurements (unittype_id, value).</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/biomaterialprop.html'>biomaterialprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Extra biomaterial properties that are not accounted for in biomaterial.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cell_line.html'>cell_line</a></td>
  <td class='detail' align='right'>9</td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cell_line_cvterm.html'>cell_line_cvterm</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cell_line_cvtermprop.html'>cell_line_cvtermprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cell_line_dbxref.html'>cell_line_dbxref</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cell_line_feature.html'>cell_line_feature</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cell_line_library.html'>cell_line_library</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cell_line_pub.html'>cell_line_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cell_line_relationship.html'>cell_line_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cell_line_synonym.html'>cell_line_synonym</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cell_lineprop.html'>cell_lineprop</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cell_lineprop_pub.html'>cell_lineprop_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/chadoprop.html'>chadoprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>This table is different from other prop tables in the database, as it is for storing information about the database itself, like schema version</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/channel.html'>channel</a></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Different array platforms can record signals from one or more channels (cDNA arrays typically use two CCD, but Affymetrix uses only one).</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/common_ancestor_cvterm.html'>common_ancestor_cvterm</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>The common ancestor of any<br>
two terms is the intersection of both terms ancestors. Two terms can<br>
have multiple common ancestors. Use total_pathdistance to get the<br>
least common ancestor</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/common_descendant_cvterm.html'>common_descendant_cvterm</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>The common descendant of<br>
any two terms is the intersection of both terms descendants. Two terms<br>
can have multiple common descendants. Use total_pathdistance to get<br>
the least common ancestor</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/contact.html'>contact</a></td>
  <td class='detail' align='right'>10</td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Model persons, institutes, groups, organizations, etc.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/contact_relationship.html'>contact_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Model relationships between contacts</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/control.html'>control</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>8</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cv.html'>cv</a></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A controlled vocabulary or ontology. A cv is<br>
composed of cvterms (AKA terms, classes, types, universals - relations<br>
and properties are also stored in cvterm) and the relationships<br>
between them.</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/cv_cvterm_count.html'>cv_cvterm_count</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>per-cv terms counts (excludes obsoletes)</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/cv_cvterm_count_with_obs.html'>cv_cvterm_count_with_obs</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>per-cv terms counts (includes obsoletes)</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/cv_leaf.html'>cv_leaf</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>the leaves of a cv are the set of terms<br>
which have no children (terms that are not the object of a<br>
relation). All cvs will have at least 1 leaf</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/cv_link_count.html'>cv_link_count</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>per-cv summary of number of<br>
links (cvterm_relationships) broken down by<br>
relationship_type. num_links is the total # of links of the specified<br>
type in which the subject_id of the link is in the named cv</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/cv_path_count.html'>cv_path_count</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>per-cv summary of number of<br>
paths (cvtermpaths) broken down by relationship_type. num_paths is the<br>
total # of paths of the specified type in which the subject_id of the<br>
path is in the named cv. See also: cv_distinct_relations</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/cv_root.html'>cv_root</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>the roots of a cv are the set of terms<br>
which have no parents (terms that are not the subject of a<br>
relation). Most cvs will have a single root, some may have &gt;1. All<br>
will have at least 1</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cvprop.html'>cvprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Additional extensible properties can be attached to a cv using this table.  A notable example would be the cv version</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cvterm.html'>cvterm</a></td>
  <td class='detail' align='right'>104</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A term, class, universal or type within an<br>
ontology or controlled vocabulary.  This table is also used for<br>
relations and properties. cvterms constitute nodes in the graph<br>
defined by the collection of cvterms and cvterm_relationships.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cvterm_dbxref.html'>cvterm_dbxref</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>In addition to the primary<br>
identifier (cvterm.dbxref_id) a cvterm can have zero or more secondary<br>
identifiers/dbxrefs, which may refer to records in external<br>
databases. The exact semantics of cvterm_dbxref are not fixed. For<br>
example: the dbxref could be a pubmed ID that is pertinent to the<br>
cvterm, or it could be an equivalent or similar term in another<br>
ontology. For example, GO cvterms are typically linked to InterPro<br>
IDs, even though the nature of the relationship between them is<br>
largely one of statistical association. The dbxref may be have data<br>
records attached in the same database instance, or it could be a<br>
"hanging" dbxref pointing to some external database. NOTE: If the<br>
desired objective is to link two cvterms together, and the nature of<br>
the relation is known and holds for all instances of the subject<br>
cvterm then consider instead using cvterm_relationship together with a<br>
well-defined relation.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cvterm_relationship.html'>cvterm_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A relationship linking two<br>
cvterms. Each cvterm_relationship constitutes an edge in the graph<br>
defined by the collection of cvterms and cvterm_relationships. The<br>
meaning of the cvterm_relationship depends on the definition of the<br>
cvterm R refered to by type_id. However, in general the definitions<br>
are such that the statement "all SUBJs REL some OBJ" is true. The<br>
cvterm_relationship statement is about the subject, not the<br>
object. For example "insect wing part_of thorax".</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cvtermpath.html'>cvtermpath</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>The reflexive transitive closure of<br>
the cvterm_relationship relation.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cvtermprop.html'>cvtermprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Additional extensible properties can be attached to a cvterm using this table. Corresponds to -AnnotationProperty- in W3C OWL format.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/cvtermsynonym.html'>cvtermsynonym</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A cvterm actually represents a<br>
distinct class or concept. A concept can be refered to by different<br>
phrases or names. In addition to the primary name (cvterm.name) there<br>
can be a number of alternative aliases or synonyms. For example, "T<br>
cell" as a synonym for "T lymphocyte".</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/db.html'>db</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A database authority. Typical databases in<br>
bioinformatics are FlyBase, GO, UniProt, NCBI, MGI, etc. The authority<br>
is generally known by this shortened form, which is unique within the<br>
bioinformatics and biomedical realm.  To Do - add support for URIs,<br>
URNs (e.g. LSIDs). We can do this by treating the URL as a URI -<br>
however, some applications may expect this to be resolvable - to be<br>
decided.</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/db_dbxref_count.html'>db_dbxref_count</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>per-db dbxref counts</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/dbxref.html'>dbxref</a></td>
  <td class='detail' align='right'>23</td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A unique, global, public, stable identifier. Not necessarily an external reference - can reference data items inside the particular chado instance being used. Typically a row in a table can be uniquely identified with a primary identifier (called dbxref_id); a table may also have secondary identifiers (in a linking table &lt;T&gt;_dbxref). A dbxref is generally written as &lt;DB&gt;:&lt;ACCESSION&gt; or as &lt;DB&gt;:&lt;ACCESSION&gt;:&lt;VERSION&gt;.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/dbxrefprop.html'>dbxrefprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Metadata about a dbxref. Note that this is not defined in the dbxref module, as it depends on the cvterm table. This table has a structure analagous to cvtermprop.</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/dfeatureloc.html'>dfeatureloc</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>12</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/eimage.html'>eimage</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/element.html'>element</a></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Represents a feature of the array. This is typically a region of the array coated or bound to DNA.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/element_relationship.html'>element_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Sometimes we want to combine measurements from multiple elements to get a composite value. Affymetrix combines many probes to form a probeset measurement, for instance.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/elementresult.html'>elementresult</a></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>An element on an array produces a measurement when hybridized to a biomaterial (traceable through quantification_id). This is the base data from which tables that actually contain data inherit.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/elementresult_relationship.html'>elementresult_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Sometimes we want to combine measurements from multiple elements to get a composite value. Affymetrix combines many probes to form a probeset measurement, for instance.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/environment.html'>environment</a></td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>The environmental component of a phenotype description.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/environment_cvterm.html'>environment_cvterm</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/expression.html'>expression</a></td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>The expression table is essentially a bridge table.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/expression_cvterm.html'>expression_cvterm</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/expression_cvtermprop.html'>expression_cvtermprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Extensible properties for<br>
expression to cvterm associations. Examples: qualifiers.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/expression_image.html'>expression_image</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/expression_pub.html'>expression_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/expressionprop.html'>expressionprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/f_loc.html'>f_loc</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/f_type.html'>f_type</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>10</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature.html'>feature</a></td>
  <td class='detail' align='right'>30</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>13</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A feature is a biological sequence or a<br>
section of a biological sequence, or a collection of such<br>
sections. Examples include genes, exons, transcripts, regulatory<br>
regions, polypeptides, protein domains, chromosome sequences, sequence<br>
variations, cross-genome match regions such as hits and HSPs and so<br>
on; see the Sequence Ontology for more. The combination of<br>
organism_id, uniquename and type_id should be unique.</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/feature_contains.html'>feature_contains</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>subject intervals contains (or is<br>
same as) object interval. transitive,reflexive</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_cvterm.html'>feature_cvterm</a></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Associate a term from a cv with a feature, for example, GO annotation.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_cvterm_dbxref.html'>feature_cvterm_dbxref</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Additional dbxrefs for an association. Rows in the feature_cvterm table may be backed up by dbxrefs. For example, a feature_cvterm association that was inferred via a protein-protein interaction may be backed by by refering to the dbxref for the alternate protein. Corresponds to the WITH column in a GO gene association file (but can also be used for other analagous associations). See http://www.geneontology.org/doc/GO.annotation.shtml#file for more details.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_cvterm_pub.html'>feature_cvterm_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Secondary pubs for an<br>
association. Each feature_cvterm association is supported by a single<br>
primary publication. Additional secondary pubs can be added using this<br>
linking table (in a GO gene association file, these corresponding to<br>
any IDs after the pipe symbol in the publications column.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_cvtermprop.html'>feature_cvtermprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Extensible properties for<br>
feature to cvterm associations. Examples: GO evidence codes;<br>
qualifiers; metadata such as the date on which the entry was curated<br>
and the source of the association. See the featureprop table for<br>
meanings of type_id, value and rank.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_dbxref.html'>feature_dbxref</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Links a feature to dbxrefs. This is for secondary identifiers; primary identifiers should use feature.dbxref_id.</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/feature_difference.html'>feature_difference</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>size of gap between two features. must be abutting or disjoint</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/feature_disjoint.html'>feature_disjoint</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>featurelocs do not meet. symmetric</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/feature_distance.html'>feature_distance</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_expression.html'>feature_expression</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_expressionprop.html'>feature_expressionprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Extensible properties for<br>
feature_expression (comments, for example). Modeled on feature_cvtermprop.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_genotype.html'>feature_genotype</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/feature_intersection.html'>feature_intersection</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>set-intersection on interval defined by featureloc. featurelocs must meet</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/feature_meets.html'>feature_meets</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>intervals have at least one<br>
interbase point in common (ie overlap OR abut). symmetric,reflexive</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/feature_meets_on_same_strand.html'>feature_meets_on_same_strand</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>as feature_meets, but<br>
featurelocs must be on the same strand. symmetric,reflexive</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_phenotype.html'>feature_phenotype</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_pub.html'>feature_pub</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Provenance. Linking table between features and publications that mention them.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_pubprop.html'>feature_pubprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Property or attribute of a feature_pub link.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_relationship.html'>feature_relationship</a></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Features can be arranged in<br>
graphs, e.g. "exon part_of transcript part_of gene"; If type is<br>
thought of as a verb, the each arc or edge makes a statement<br>
[Subject Verb Object]. The object can also be thought of as parent<br>
(containing feature), and subject as child (contained feature or<br>
subfeature). We include the relationship rank/order, because even<br>
though most of the time we can order things implicitly by sequence<br>
coordinates, we can not always do this - e.g. transpliced genes. It is also<br>
useful for quickly getting implicit introns.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_relationship_pub.html'>feature_relationship_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Provenance. Attach optional evidence to a feature_relationship in the form of a publication.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_relationshipprop.html'>feature_relationshipprop</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Extensible properties<br>
for feature_relationships. Analagous structure to featureprop. This<br>
table is largely optional and not used with a high frequency. Typical<br>
scenarios may be if one wishes to attach additional data to a<br>
feature_relationship - for example to say that the<br>
feature_relationship is only true in certain contexts.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_relationshipprop_pub.html'>feature_relationshipprop_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Provenance for feature_relationshipprop.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/feature_synonym.html'>feature_synonym</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Linking table between feature and synonym.</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/feature_union.html'>feature_union</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>set-union on interval defined by featureloc. featurelocs must meet</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/featureloc.html'>featureloc</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>12</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>The location of a feature relative to<br>
another feature. Important: interbase coordinates are used. This is<br>
vital as it allows us to represent zero-length features e.g. splice<br>
sites, insertion points without an awkward fuzzy system. Features<br>
typically have exactly ONE location, but this need not be the<br>
case. Some features may not be localized (e.g. a gene that has been<br>
characterized genetically but no sequence or molecular information is<br>
available). Note on multiple locations: Each feature can have 0 or<br>
more locations. Multiple locations do NOT indicate non-contiguous<br>
locations (if a feature such as a transcript has a non-contiguous<br>
location, then the subfeatures such as exons should always be<br>
manifested). Instead, multiple featurelocs for a feature designate<br>
alternate locations or grouped locations; for instance, a feature<br>
designating a blast hit or hsp will have two locations, one on the<br>
query feature, one on the subject feature. Features representing<br>
sequence variation could have alternate locations instantiated on a<br>
feature on the mutant strain. The column:rank is used to<br>
differentiate these different locations. Reflexive locations should<br>
never be stored - this is for -proper- (i.e. non-self) locations only; nothing should be located relative to itself.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/featureloc_pub.html'>featureloc_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Provenance of featureloc. Linking table between featurelocs and publications that mention them.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/featuremap.html'>featuremap</a></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/featuremap_pub.html'>featuremap_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/featurepos.html'>featurepos</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/featureprop.html'>featureprop</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A feature can have any number of slot-value property tags attached to it. This is an alternative to hardcoding a list of columns in the relational schema, and is completely extensible.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/featureprop_pub.html'>featureprop_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Provenance. Any featureprop assignment can optionally be supported by a publication.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/featurerange.html'>featurerange</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>8</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>In cases where the start and end of a mapped feature is a range, leftendf and rightstartf are populated. leftstartf_id, leftendf_id, rightstartf_id, rightendf_id are the ids of features with respect to which the feature is being mapped. These may be cytological bands.</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/featureset_meets.html'>featureset_meets</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/fnr_type.html'>fnr_type</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>10</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/fp_key.html'>fp_key</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/genotype.html'>genotype</a></td>
  <td class='detail' align='right'>8</td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Genetic context. A genotype is defined by a collection of features, mutations, balancers, deficiencies, haplotype blocks, or engineered constructs.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/genotypeprop.html'>genotypeprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/gff3atts.html'>gff3atts</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/gff3view.html'>gff3view</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>12</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/gffatts.html'>gffatts</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/intron_combined_view.html'>intron_combined_view</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>8</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/intronloc_view.html'>intronloc_view</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/library.html'>library</a></td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>8</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/library_cvterm.html'>library_cvterm</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>The table library_cvterm links a library to controlled vocabularies which describe the library.  For instance, there might be a link to the anatomy cv for "head" or "testes" for a head or testes library.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/library_dbxref.html'>library_dbxref</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/library_feature.html'>library_feature</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>library_feature links a library to the clones which are contained in the library.  Examples of such linked features might be "cDNA_clone" or  "genomic_clone".</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/library_pub.html'>library_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/library_synonym.html'>library_synonym</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/libraryprop.html'>libraryprop</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/libraryprop_pub.html'>libraryprop_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/magedocumentation.html'>magedocumentation</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/mageml.html'>mageml</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>This table is for storing extra bits of MAGEml in a denormalized form. More normalization would require many more tables.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experiment.html'>nd_experiment</a></td>
  <td class='detail' align='right'>9</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experiment_contact.html'>nd_experiment_contact</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experiment_dbxref.html'>nd_experiment_dbxref</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Cross-reference experiment to accessions, images, etc</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experiment_genotype.html'>nd_experiment_genotype</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Linking table: experiments to the genotypes they produce. There is a one-to-one relationship between an experiment and a genotype since each genotype record should point to one experiment. Add a new experiment_id for each genotype record.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experiment_phenotype.html'>nd_experiment_phenotype</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Linking table: experiments to the phenotypes they produce. There is a one-to-one relationship between an experiment and a phenotype since each phenotype record should point to one experiment. Add a new experiment_id for each phenotype record.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experiment_project.html'>nd_experiment_project</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experiment_protocol.html'>nd_experiment_protocol</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Linking table: experiments to the protocols they involve.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experiment_pub.html'>nd_experiment_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Linking nd_experiment(s) to publication(s)</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experiment_stock.html'>nd_experiment_stock</a></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Part of a stock or a clone of a stock that is used in an experiment</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experiment_stock_dbxref.html'>nd_experiment_stock_dbxref</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Cross-reference experiment_stock to accessions, images, etc</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experiment_stockprop.html'>nd_experiment_stockprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Property/value associations for experiment_stocks. This table can store the properties such as treatment</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_experimentprop.html'>nd_experimentprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_geolocation.html'>nd_geolocation</a></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>The geo-referencable location of the stock. NOTE: This entity is subject to change as a more general and possibly more OpenGIS-compliant geolocation module may be introduced into Chado.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_geolocationprop.html'>nd_geolocationprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Property/value associations for geolocations. This table can store the properties such as location and environment</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_protocol.html'>nd_protocol</a></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A protocol can be anything that is done as part of the experiment.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_protocol_reagent.html'>nd_protocol_reagent</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_protocolprop.html'>nd_protocolprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Property/value associations for protocol.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_reagent.html'>nd_reagent</a></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A reagent such as a primer, an enzyme, an adapter oligo, a linker oligo. Reagents are used in genotyping experiments, or in any other kind of experiment.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_reagent_relationship.html'>nd_reagent_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Relationships between reagents. Some reagents form a group. i.e., they are used all together or not at all. Examples are adapter/linker/enzyme experiment reagents.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/nd_reagentprop.html'>nd_reagentprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/organism.html'>organism</a></td>
  <td class='detail' align='right'>9</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>The organismal taxonomic<br>
classification. Note that phylogenies are represented using the<br>
phylogeny module, and taxonomies can be represented using the cvterm<br>
module or the phylogeny module.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/organism_dbxref.html'>organism_dbxref</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/organismprop.html'>organismprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Tag-value properties - follows standard chado model.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phendesc.html'>phendesc</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A summary of a _set_ of phenotypic statements for any one gcontext made in any one publication.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phenotype.html'>phenotype</a></td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>8</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A phenotypic statement, or a single<br>
atomic phenotypic observation, is a controlled sentence describing<br>
observable effects of non-wild type function. E.g. Obs=eye, attribute=color, cvalue=red.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phenotype_comparison.html'>phenotype_comparison</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>8</td>
  <td class='detail' align='right'>9</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Comparison of phenotypes e.g., genotype1/environment1/phenotype1 "non-suppressible" with respect to genotype2/environment2/phenotype2.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phenotype_comparison_cvterm.html'>phenotype_comparison_cvterm</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phenotype_cvterm.html'>phenotype_cvterm</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phenstatement.html'>phenstatement</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Phenotypes are things like "larval lethal".  Phenstatements are things like "dpp-1 is recessive larval lethal". So essentially phenstatement is a linking table expressing the relationship between genotype, environment, and phenotype.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phylonode.html'>phylonode</a></td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>9</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>This is the most pervasive<br>
       element in the phylogeny module, cataloging the "phylonodes" of<br>
       tree graphs. Edges are implied by the parent_phylonode_id<br>
       reflexive closure. For all nodes in a nested set implementation the left and right index will be *between* the parents left and right indexes.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phylonode_dbxref.html'>phylonode_dbxref</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>For example, for orthology, paralogy group identifiers; could also be used for NCBI taxonomy; for sequences, refer to phylonode_feature, feature associated dbxrefs.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phylonode_organism.html'>phylonode_organism</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>This linking table should only be used for nodes in taxonomy trees; it provides a mapping between the node and an organism. One node can have zero or one organisms, one organism can have zero or more nodes (although typically it should only have one in the standard NCBI taxonomy tree).</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phylonode_pub.html'>phylonode_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phylonode_relationship.html'>phylonode_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>This is for <br>
relationships that are not strictly hierarchical; for example,<br>
horizontal gene transfer. Most phylogenetic trees are strictly<br>
hierarchical, nevertheless it is here for completeness.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phylonodeprop.html'>phylonodeprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phylotree.html'>phylotree</a></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Global anchor for phylogenetic tree.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/phylotree_pub.html'>phylotree_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Tracks citations global to the tree e.g. multiple sequence alignment supporting tree construction.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/project.html'>project</a></td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/project_contact.html'>project_contact</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Linking project(s) to contact(s)</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/project_pub.html'>project_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Linking project(s) to publication(s)</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/project_relationship.html'>project_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A project can be composed of several smaller scale projects</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/projectprop.html'>projectprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/protein_coding_gene.html'>protein_coding_gene</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>13</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/protocol.html'>protocol</a></td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>9</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Procedural notes on how data was prepared and processed.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/protocolparam.html'>protocolparam</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Parameters related to a<br>
protocol. For example, if the protocol is a soak, this might include attributes of bath temperature and duration.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/pub.html'>pub</a></td>
  <td class='detail' align='right'>41</td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>14</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A documented provenance artefact - publications,<br>
documents, personal communication.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/pub_dbxref.html'>pub_dbxref</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Handle links to repositories,<br>
e.g. Pubmed, Biosis, zoorec, OCLC, Medline, ISSN, coden...</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/pub_relationship.html'>pub_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Handle relationships between<br>
publications, e.g. when one publication makes others obsolete, when one<br>
publication contains errata with respect to other publication(s), or<br>
when one publication also appears in another pub.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/pubauthor.html'>pubauthor</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>7</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>An author for a publication. Note the denormalisation (hence lack of _ in table name) - this is deliberate as it is in general too hard to assign IDs to authors.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/pubprop.html'>pubprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Property-value pairs for a pub. Follows standard chado pattern.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/quantification.html'>quantification</a></td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>8</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Quantification is the transformation of an image acquisition to numeric data. This typically involves statistical procedures.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/quantification_relationship.html'>quantification_relationship</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>There may be multiple rounds of quantification, this allows us to keep an audit trail of what values went where.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/quantificationprop.html'>quantificationprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Extra quantification properties that are not accounted for in quantification.</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/stats_paths_to_root.html'>stats_paths_to_root</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>per-cvterm statistics on its<br>
placement in the DAG relative to the root. There may be multiple paths<br>
from any term to the root. This gives the total number of paths, and<br>
the average minimum and maximum distances. Here distance is defined by<br>
cvtermpath.pathdistance</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stock.html'>stock</a></td>
  <td class='detail' align='right'>9</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>8</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Any stock can be globally identified by the<br>
combination of organism, uniquename and stock type. A stock is the physical entities, either living or preserved, held by collections. Stocks belong to a collection; they have IDs, type, organism, description and may have a genotype.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stock_cvterm.html'>stock_cvterm</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>stock_cvterm links a stock to cvterms. This is for secondary cvterms; primary cvterms should use stock.type_id.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stock_cvtermprop.html'>stock_cvtermprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Extensible properties for<br>
stock to cvterm associations. Examples: GO evidence codes;<br>
qualifiers; metadata such as the date on which the entry was curated<br>
and the source of the association. See the stockprop table for<br>
meanings of type_id, value and rank.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stock_dbxref.html'>stock_dbxref</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>stock_dbxref links a stock to dbxrefs. This is for secondary identifiers; primary identifiers should use stock.dbxref_id.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stock_dbxrefprop.html'>stock_dbxrefprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A stock_dbxref can have any number of<br>
slot-value property tags attached to it. This is useful for storing properties related to dbxref annotations of stocks, such as evidence codes, and references, and metadata, such as create/modify dates. This is an alternative to<br>
hardcoding a list of columns in the relational schema, and is<br>
completely extensible. There is a unique constraint, stock_dbxrefprop_c1, for<br>
the combination of stock_dbxref_id, rank, and type_id. Multivalued property-value pairs must be differentiated by rank.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stock_genotype.html'>stock_genotype</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Simple table linking a stock to<br>
a genotype. Features with genotypes can be linked to stocks thru feature_genotype -&gt; genotype -&gt; stock_genotype -&gt; stock.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stock_pub.html'>stock_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Provenance. Linking table between stocks and, for example, a stocklist computer file.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stock_relationship.html'>stock_relationship</a></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stock_relationship_cvterm.html'>stock_relationship_cvterm</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>For germplasm maintenance and pedigree data, stock_relationship. type_id will record cvterms such as "is a female parent of", "a parent for mutation", "is a group_id of", "is a source_id of", etc The cvterms for higher categories such as "generative", "derivative" or "maintenance" can be stored in table stock_relationship_cvterm</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stock_relationship_pub.html'>stock_relationship_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Provenance. Attach optional evidence to a stock_relationship in the form of a publication.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stockcollection.html'>stockcollection</a></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>The lab or stock center distributing the stocks in their collection.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stockcollection_stock.html'>stockcollection_stock</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>stockcollection_stock links<br>
a stock collection to the stocks which are contained in the collection.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stockcollectionprop.html'>stockcollectionprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>The table stockcollectionprop<br>
contains the value of the stock collection such as website/email URLs;<br>
the value of the stock collection order URLs.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stockprop.html'>stockprop</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A stock can have any number of<br>
slot-value property tags attached to it. This is an alternative to<br>
hardcoding a list of columns in the relational schema, and is<br>
completely extensible. There is a unique constraint, stockprop_c1, for<br>
the combination of stock_id, rank, and type_id. Multivalued property-value pairs must be differentiated by rank.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/stockprop_pub.html'>stockprop_pub</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>Provenance. Any stockprop assignment can optionally be supported by a publication.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/study.html'>study</a></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/study_assay.html'>study_assay</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/studydesign.html'>studydesign</a></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/studydesignprop.html'>studydesignprop</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/studyfactor.html'>studyfactor</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/studyfactorvalue.html'>studyfactorvalue</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/studyprop.html'>studyprop</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'>5</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/studyprop_feature.html'>studyprop_feature</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/synonym.html'>synonym</a></td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>4</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A synonym for a feature. One feature can have multiple synonyms, and the same synonym can apply to multiple features.</td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/tableinfo.html'>tableinfo</a></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>8</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'></td>
  </tr>
 <tr class='tbl' valign='top'>  <td class='detail'><a href='tables/treatment.html'>treatment</a></td>
  <td class='detail' align='right'>1</td>
  <td class='detail' align='right'>3</td>
  <td class='detail' align='right'>6</td>
  <td class='detail' align='right'>0</td>
  <td class='comment detail'>A biomaterial may undergo multiple<br>
treatments. Examples of treatments: apoxia, fluorophore and biotin labeling.</td>
  </tr>
 <tr class='view' valign='top'>  <td class='detail'><a href='tables/type_feature_count.html'>type_feature_count</a></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'></td>
  <td class='detail' align='right'>2</td>
  <td class='detail' align='right'><span title='Views contain no real rows'>view</span></td>
  <td class='comment detail'>per-feature-type feature counts</td>
  </tr>
  <tr>
    <td class='detail'>&nbsp;</td>
    <td class='detail'>&nbsp;</td>
    <td class='detail'>&nbsp;</td>
    <td class='detail'>&nbsp;</td>
    <td class='detail'>&nbsp;</td>
    <td class='comment detail'>&nbsp;</td>
  </tr>
  <tr class='tbl'>
    <td class='detail'><b>173 Tables</b></td>
    <td class='detail'>&nbsp;</td>
    <td class='detail'>&nbsp;</td>
    <td class='detail' align='right'><b>852</b></td>
    <td class='detail' align='right'><b>0</b></td>
    <td class='comment detail'>&nbsp;</td>
  </tr>
  <tr class='view'>
    <td class='detail'><b>32 Views</b></td>
    <td class='detail'>&nbsp;</td>
    <td class='detail'>&nbsp;</td>
    <td class='detail' align='right'><b>162</b></td>
    <td class='detail'>&nbsp;</td>
    <td class='comment detail'>&nbsp;</td>
  </tr>
</table>
</div>
</body>
</html>
