<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01 Transitional//EN' 'http://www.w3.org/TR/html4/loose.dtd'>
<html>
<head>
  <!-- SchemaSpy rev 590 -->
  <title>SchemaSpy - Table dicty_chado.public.featureloc</title>
  <link rel=stylesheet href='../schemaSpy.css' type='text/css'>
  <meta HTTP-EQUIV='Content-Type' CONTENT='text/html; charset=ISO-8859-1'>
  <SCRIPT LANGUAGE='JavaScript' TYPE='text/javascript' SRC='../jquery.js'></SCRIPT>
  <SCRIPT LANGUAGE='JavaScript' TYPE='text/javascript' SRC='../schemaSpy.js'></SCRIPT>
  <SCRIPT LANGUAGE='JavaScript' TYPE='text/javascript'>
    table='featureloc';
  </SCRIPT>
</head>
<body>
<table id='headerHolder' cellspacing='0' cellpadding='0'><tr><td>
<div id='header'>
 <ul>
  <li><a href='../index.html' title='All tables and views in the schema'>Tables</a></li>
  <li><a href='../relationships.html' title='Diagram of table relationships'>Relationships</a></li>
  <li><a href='../constraints.html' title='Useful for diagnosing error messages that just give constraint name or number'>Constraints</a></li>
  <li><a href='../anomalies.html' title="Things that might not be quite right">Anomalies</a></li>
  <li><a href='../columns.byTable.html' title="All of the columns in the schema">Columns</a></li>
  <li><a href='http://sourceforge.net/donate/index.php?group_id=137197' title='Please help keep SchemaSpy alive' target='_blank'>Donate</a></li>
 </ul>
</div>
</td></tr></table>
<div class='content' style='clear:both;'>
<table width='100%' border='0' cellpadding='0'>
 <tr>
  <td class='heading' valign='middle'><span class='header'>Table <span title='Database'>dicty_chado</span>.<span title='Schema'>public</span>.<span title='Table'>featureloc</span></span><div style='padding: 0px 4px;'>The location of a feature relative to<br>
another feature. Important: interbase coordinates are used. This is<br>
vital as it allows us to represent zero-length features e.g. splice<br>
sites, insertion points without an awkward fuzzy system. Features<br>
typically have exactly ONE location, but this need not be the<br>
case. Some features may not be localized (e.g. a gene that has been<br>
characterized genetically but no sequence or molecular information is<br>
available). Note on multiple locations: Each feature can have 0 or<br>
more locations. Multiple locations do NOT indicate non-contiguous<br>
locations (if a feature such as a transcript has a non-contiguous<br>
location, then the subfeatures such as exons should always be<br>
manifested). Instead, multiple featurelocs for a feature designate<br>
alternate locations or grouped locations; for instance, a feature<br>
designating a blast hit or hsp will have two locations, one on the<br>
query feature, one on the subject feature. Features representing<br>
sequence variation could have alternate locations instantiated on a<br>
feature on the mutant strain. The column:rank is used to<br>
differentiate these different locations. Reflexive locations should<br>
never be stored - this is for -proper- (i.e. non-self) locations only; nothing should be located relative to itself.</div><p>
</td>
  <td class='heading' align='right' valign='top' title='John Currier - Creator of Cool Tools'><span class='indent'>Generated by</span><br><span class='indent'><span class='signature'><a href='http://schemaspy.sourceforge.net' target='_blank'>SchemaSpy</a></span></span></td>
 </tr>
</table>
<table width='100%' border='0'>
<tr valign='top'><td class='container' align='left' valign='top'>
<form name='options' action=''>
 <label for='showRelatedCols'><input type=checkbox id='showRelatedCols'>Related columns</label>
 <label for='showConstNames'><input type=checkbox id='showConstNames'>Constraints</label>
 <label for='showComments'><input type=checkbox checked id='showComments'>Comments</label>
 <label for='showLegend'><input type=checkbox checked id='showLegend'>Legend</label>
</form>
</td><td class='container' rowspan='2' align='right' valign='top'>
 <table class='legend' border='0'>
  <tr>
   <td class='dataTable' valign='bottom'>Legend:</td>
   <td class='container' align='right' valign='top'><a href='http://sourceforge.net' target='_blank'><img src='http://sourceforge.net/sflogo.php?group_id=137197&amp;type=1' alt='SourceForge.net' border='0' height='31' width='88'></a></td>
  </tr>
  <tr><td class='container' colspan='2'>
   <table class='dataTable' border='1'>
    <tbody>
    <tr><td class='primaryKey'>Primary key columns</td></tr>
    <tr><td class='indexedColumn'>Columns with indexes</td></tr>
    <tr class='impliedRelationship'><td class='detail'><span class='impliedRelationship'>Implied relationships</span></td></tr>
    <tr><td class='excludedColumn'>Excluded column relationships</td></tr>
    <tr><td class='legendDetail'>&lt; <em>n</em> &gt; number of related tables</td></tr>
   </table>
  </td></tr>
 </table>
<div style="margin-right: 2pt;">
<script type="text/javascript"><!--
google_ad_client = "pub-9598353634003340";
google_ad_channel ="SchemaSpy-generated";
google_ad_width = 234;
google_ad_height = 60;
google_ad_format = "234x60_as";
google_ad_type = "text";
google_color_border = "9bab96";
google_color_link = "489148";
google_color_text = "000000";
//-->
</script>
<script type="text/javascript"
src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</div>
&nbsp;
</td><tr valign='top'><td class='container' align='left' valign='top'>
<a name='columns'></a>
<table id='columns' class='dataTable' border='1' rules='groups'>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup>
<colgroup class='comment'>
<thead align='left'>
<tr>
  <th>Column</th>
  <th>Type</th>
  <th>Size</th>
  <th title='Are nulls allowed?'>Nulls</th>
  <th title='Is column automatically updated?'>Auto</th>
  <th title='Default value'>Default</th>
  <th title='Columns in tables that reference this column'><span class='notSortedByColumn'>Children</span></th>
  <th title='Columns in tables that are referenced by this column'><span class='notSortedByColumn'>Parents</span></th>
  <th title='Comments' class='comment'><span class='notSortedByColumn'>Comments</span></th>
</tr>
</thead>
<tbody valign='top'>
<tr class='odd'>
 <td class='primaryKey' title='Primary Key'>featureloc_id</td>
 <td class='detail'>serial</td>
 <td class='detail' align='right'>10</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='center' title='Automatically updated by the database'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='right'>nextval('featureloc_featureloc_id_seq'::regclass)</td>
 <td class='detail'>
  <table border='0' width='100%' cellspacing='0' cellpadding='0'>
   <tr class='relative odd' valign='top'>
    <td class='relatedTable detail' title="featureloc_pub.featureloc_id refs featureloc.featureloc_id via featureloc_pub_featureloc_id_fkey"><a href='featureloc_pub.html'>featureloc_pub</a><span class='relatedKey'>.featureloc_id</span>
    </td>
    <td class='constraint detail'>featureloc_pub_featureloc_id_fkey<span title='Cascade on delete:
 Deletion of parent deletes child'>&nbsp;C</span></td>
   </tr>
  </table>
</td>
 <td class='detail'> </td>
 <td class='comment detail'></td>
</tr>
<tr class='even'>
 <td class='indexedColumn' title='Indexed'>feature_id</td>
 <td class='detail'>int4</td>
 <td class='detail' align='right'>10</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='center'></td>
 <td class='detail'></td>
 <td class='detail'></td>
 <td class='detail'>
  <table border='0' width='100%' cellspacing='0' cellpadding='0'>
   <tr class='relative even' valign='top'>
    <td class='relatedTable detail' title="featureloc.feature_id refs feature.feature_id via featureloc_feature_id_fkey"><a href='feature.html'>feature</a><span class='relatedKey'>.feature_id</span>
    </td>
    <td class='constraint detail'>featureloc_feature_id_fkey<span title='Cascade on delete:
 Deletion of parent deletes child'>&nbsp;C</span></td>
   </tr>
  </table>
 </td>
 <td class='comment detail'>The feature that is being located. Any feature can have zero or more featurelocs.</td>
</tr>
<tr class='odd'>
 <td class='indexedColumn' title='Indexed'>srcfeature_id</td>
 <td class='detail'>int4</td>
 <td class='detail' align='right'>10</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'>
  <table border='0' width='100%' cellspacing='0' cellpadding='0'>
   <tr class='relative odd' valign='top'>
    <td class='relatedTable detail' title="featureloc.srcfeature_id refs feature.feature_id via featureloc_srcfeature_id_fkey"><a href='feature.html'>feature</a><span class='relatedKey'>.feature_id</span>
    </td>
    <td class='constraint detail'>featureloc_srcfeature_id_fkey<span title='Null on delete:
 Foreign key to parent set to NULL when parent deleted'>&nbsp;N</span></td>
   </tr>
  </table>
 </td>
 <td class='comment detail'>The source feature which this location is relative to. Every location is relative to another feature (however, this column is nullable, because the srcfeature may not be known). All locations are -proper- that is, nothing should be located relative to itself. No cycles are allowed in the featureloc graph.</td>
</tr>
<tr class='even'>
 <td class='indexedColumn' title='Indexed'>fmin</td>
 <td class='detail'>int4</td>
 <td class='detail' align='right'>10</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'>The leftmost/minimal boundary in the linear range represented by the featureloc. Sometimes (e.g. in Bioperl) this is called -start- although this is confusing because it does not necessarily represent the 5-prime coordinate. Important: This is space-based (interbase) coordinates, counting from zero. To convert this to the leftmost position in a base-oriented system (eg GFF, Bioperl), add 1 to fmin.</td>
</tr>
<tr class='odd'>
 <td class='detail'>is_fmin_partial</td>
 <td class='detail'>bool</td>
 <td class='detail' align='right'>1</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>false</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'>This is typically<br>
false, but may be true if the value for column:fmin is inaccurate or<br>
the leftmost part of the range is unknown/unbounded.</td>
</tr>
<tr class='even'>
 <td class='indexedColumn' title='Indexed'>fmax</td>
 <td class='detail'>int4</td>
 <td class='detail' align='right'>10</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'>The rightmost/maximal boundary in the linear range represented by the featureloc. Sometimes (e.g. in bioperl) this is called -end- although this is confusing because it does not necessarily represent the 3-prime coordinate. Important: This is space-based (interbase) coordinates, counting from zero. No conversion is required to go from fmax to the rightmost coordinate in a base-oriented system that counts from 1 (e.g. GFF, Bioperl).</td>
</tr>
<tr class='odd'>
 <td class='detail'>is_fmax_partial</td>
 <td class='detail'>bool</td>
 <td class='detail' align='right'>1</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>false</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'>This is typically<br>
false, but may be true if the value for column:fmax is inaccurate or<br>
the rightmost part of the range is unknown/unbounded.</td>
</tr>
<tr class='even'>
 <td class='detail'>strand</td>
 <td class='detail'>int2</td>
 <td class='detail' align='right'>5</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'>The orientation/directionality of the<br>
location. Should be 0, -1 or +1.</td>
</tr>
<tr class='odd'>
 <td class='detail'>phase</td>
 <td class='detail'>int4</td>
 <td class='detail' align='right'>10</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'>Phase of translation with<br>
respect to srcfeature_id.<br>
Values are 0, 1, 2. It may not be possible to manifest this column for<br>
some features such as exons, because the phase is dependant on the<br>
spliceform (the same exon can appear in multiple spliceforms). This column is mostly useful for predicted exons and CDSs.</td>
</tr>
<tr class='even'>
 <td class='detail'>residue_info</td>
 <td class='detail'>text</td>
 <td class='detail' align='right'>2147483647</td>
 <td class='detail' align='center' title='nullable'>&nbsp;&radic;&nbsp;</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>null</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'>Alternative residues,<br>
when these differ from feature.residues. For instance, a SNP feature<br>
located on a wild and mutant protein would have different alternative residues.<br>
for alignment/similarity features, the alternative residues is used to<br>
represent the alignment string (CIGAR format). Note on variation<br>
features; even if we do not want to instantiate a mutant<br>
chromosome/contig feature, we can still represent a SNP etc with 2<br>
locations, one (rank 0) on the genome, the other (rank 1) would have<br>
most fields null, except for alternative residues.</td>
</tr>
<tr class='odd'>
 <td class='indexedColumn' title='Indexed'>locgroup</td>
 <td class='detail'>int4</td>
 <td class='detail' align='right'>10</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>0</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'>This is used to manifest redundant,<br>
derivable extra locations for a feature. The default locgroup=0 is<br>
used for the DIRECT location of a feature. Important: most Chado users may<br>
never use featurelocs WITH logroup &gt; 0. Transitively derived locations<br>
are indicated with locgroup &gt; 0. For example, the position of an exon on<br>
a BAC and in global chromosome coordinates. This column is used to<br>
differentiate these groupings of locations. The default locgroup 0<br>
is used for the main or primary location, from which the others can be<br>
derived via coordinate transformations. Another example of redundant<br>
locations is storing ORF coordinates relative to both transcript and<br>
genome. Redundant locations open the possibility of the database<br>
getting into inconsistent states; this schema gives us the flexibility<br>
of both warehouse instantiations with redundant locations (easier for<br>
querying) and management instantiations with no redundant<br>
locations. An example of using both locgroup and rank: imagine a<br>
feature indicating a conserved region between the chromosomes of two<br>
different species. We may want to keep redundant locations on both<br>
contigs and chromosomes. We would thus have 4 locations for the single<br>
conserved region feature - two distinct locgroups (contig level and<br>
chromosome level) and two distinct ranks (for the two species).</td>
</tr>
<tr class='even'>
 <td class='indexedColumn' title='Indexed'>rank</td>
 <td class='detail'>int4</td>
 <td class='detail' align='right'>10</td>
 <td class='detail' align='center'></td>
 <td class='detail' align='center'></td>
 <td class='detail' align='right'>0</td>
 <td class='detail'></td>
 <td class='detail'> </td>
 <td class='comment detail'>Used when a feature has &gt;1<br>
location, otherwise the default rank 0 is used. Some features (e.g.<br>
blast hits and HSPs) have two locations - one on the query and one on<br>
the subject. Rank is used to differentiate these. Rank=0 is always<br>
used for the query, Rank=1 for the subject. For multiple alignments,<br>
assignment of rank is arbitrary. Rank is also used for<br>
sequence_variant features, such as SNPs. Rank=0 indicates the wildtype<br>
(or baseline) feature, Rank=1 indicates the mutant (or compared) feature.</td>
</tr>
</table>
<p title='12 columns'>Table contained 0 rows at Fri Jun 06 11:08 CDT 2014<p/>
</td></tr></table>
<div class='indent'>
<b>Indexes:</b>
<table class='dataTable' border='1' rules='groups'><colgroup><colgroup><colgroup><colgroup>
<thead>
 <tr>
  <th>Column(s)</th>
  <th>Type</th>
  <th>Sort</th>
  <th class='constraint' style='text-align:left;'>Constraint Name</th>
 </tr>
</thead>
<tbody>
 <tr>
  <td class='primaryKey'>featureloc_id</td>
  <td class='detail'>Primary key</td>
  <td class='detail' style='text-align:left;'><span title='Ascending'>Asc</span></td>
  <td class='constraint' style='text-align:left;'>featureloc_pkey</td>
 </tr>
 <tr>
  <td class='indexedColumn'>feature_id + locgroup + rank</td>
  <td class='detail'>Must be unique</td>
  <td class='detail' style='text-align:left;'><span title='Ascending'>Asc</span>/<span title='Ascending'>Asc</span>/<span title='Ascending'>Asc</span></td>
  <td class='constraint' style='text-align:left;'>featureloc_c1</td>
 </tr>
 <tr>
  <td class='indexedColumn'>feature_id</td>
  <td class='detail'>Performance</td>
  <td class='detail' style='text-align:left;'><span title='Ascending'>Asc</span></td>
  <td class='constraint' style='text-align:left;'>featureloc_idx1</td>
 </tr>
 <tr>
  <td class='indexedColumn'>srcfeature_id</td>
  <td class='detail'>Performance</td>
  <td class='detail' style='text-align:left;'><span title='Ascending'>Asc</span></td>
  <td class='constraint' style='text-align:left;'>featureloc_idx2</td>
 </tr>
 <tr>
  <td class='indexedColumn'>srcfeature_id + fmin + fmax</td>
  <td class='detail'>Performance</td>
  <td class='detail' style='text-align:left;'><span title='Ascending'>Asc</span>/<span title='Ascending'>Asc</span>/<span title='Ascending'>Asc</span></td>
  <td class='constraint' style='text-align:left;'>featureloc_idx3</td>
 </tr>
</table>
</div>
<table width='100%' border='0'><tr><td class='container'>
<br><form action='get'><b>Close relationships</b><span class='degrees' id='degrees' title='Detail diminishes with increased separation from featureloc'>
&nbsp;within <label for='oneDegree'><input type='radio' name='degrees' id='oneDegree' checked>one</label>  <label for='twoDegrees'><input type='radio' name='degrees' id='twoDegrees'>two degrees</label> of separation</span><b>:</b></form>
<map id="oneDegreeRelationshipsDiagram" name="oneDegreeRelationshipsDiagram">
<area shape="rect" id="node1" href="featureloc.html" title="featureloc" alt="" coords="187,27,440,361">
<area shape="rect" id="node2" href="feature.html" title="feature" alt="" coords="5,63,143,413">
<area shape="rect" id="node3" href="featureloc_pub.html" title="featureloc_pub" alt="" coords="484,5,629,129">
</map>
  <a name='diagram'><img id='oneDegreeImg' src='../diagrams/featureloc.1degree.png' usemap='#oneDegreeRelationshipsDiagram' class='diagram' border='0' alt='' align='left'></a>
<map id="twoDegreesRelationshipsDiagram" name="twoDegreesRelationshipsDiagram">
<area shape="rect" id="node1" href="analysisfeature.html" title="analysisfeature" alt="" coords="617,933,745,1012">
<area shape="rect" id="node2" href="feature.html" title="feature" alt="" coords="373,813,511,1164">
<area shape="rect" id="node5" href="cvterm.html" title="cvterm" alt="" coords="185,1195,329,1273">
<area shape="rect" id="node6" href="dbxref.html" title="dbxref" alt="" coords="5,339,141,417">
<area shape="rect" id="node8" href="organism.html" title="organism" alt="" coords="199,857,315,936">
<area shape="rect" id="node3" href="cell_line_feature.html" title="cell_line_feature" alt="" coords="613,1413,749,1492">
<area shape="rect" id="node4" href="pub.html" title="pub" alt="" coords="374,1605,510,1684">
<area shape="rect" id="node7" href="element.html" title="element" alt="" coords="617,101,745,180">
<area shape="rect" id="node9" href="feature_cvterm.html" title="feature_cvterm" alt="" coords="617,1893,745,1972">
<area shape="rect" id="node10" href="feature_dbxref.html" title="feature_dbxref" alt="" coords="620,5,743,84">
<area shape="rect" id="node11" href="feature_expression.html" title="feature_expression" alt="" coords="605,1509,758,1588">
<area shape="rect" id="node12" href="feature_genotype.html" title="feature_genotype" alt="" coords="610,197,753,276">
<area shape="rect" id="node13" href="feature_phenotype.html" title="feature_phenotype" alt="" coords="606,1029,757,1108">
<area shape="rect" id="node14" href="feature_pub.html" title="feature_pub" alt="" coords="617,1605,745,1684">
<area shape="rect" id="node15" href="feature_relationship.html" title="feature_relationship" alt="" coords="603,1125,760,1204">
<area shape="rect" id="node16" href="feature_synonym.html" title="feature_synonym" alt="" coords="611,1701,752,1780">
<area shape="rect" id="node17" href="featureloc.html" title="featureloc" alt="" coords="555,294,808,627">
<area shape="rect" id="node18" href="featureloc_pub.html" title="featureloc_pub" alt="" coords="852,1093,997,1217">
<area shape="rect" id="node19" href="featurepos.html" title="featurepos" alt="" coords="623,645,739,724">
<area shape="rect" id="node20" href="featureprop.html" title="featureprop" alt="" coords="617,1797,745,1876">
<area shape="rect" id="node21" href="featurerange.html" title="featurerange" alt="" coords="623,741,739,820">
<area shape="rect" id="node22" href="library_feature.html" title="library_feature" alt="" coords="620,837,743,916">
<area shape="rect" id="node23" href="phylonode.html" title="phylonode" alt="" coords="617,1221,745,1300">
<area shape="rect" id="node24" href="studyprop_feature.html" title="studyprop_feature" alt="" coords="608,1317,755,1396">
</map>

  <a name='diagram'><img id='twoDegreesImg' src='../diagrams/featureloc.2degrees.png' usemap='#twoDegreesRelationshipsDiagram' class='diagram' border='0' alt='' align='left'></a>
</td></tr></table>
</div>
</body>
</html>
